frontend/src/locales/de.json

```diff
@@ -127,7 +127,7 @@
       },
       "printReact": {
         "downloadNuxtPdfListItem": {
```

Comment:
wieso heissen hier beide nuxt?

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/3036/commits/abe5bd69b9ce833ec7f3432f27a82c851004e91e